pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -21,6 +21,7 @@ func ProvideService(cfg *setting.Cfg, usageStats usagestats.Service) *OSSAccessC
 		scopeResolver: accesscontrol.NewScopeResolver(),
 	}
 	s.registerUsageMetrics()
+	accesscontrol.InitPerms()
```

Comment:
yea, but I am not sure that this is correct to be here

Reply:
this is just that I think something big is happening because this is initiated inside of the serviceProvider